app/src/test/java/com/android/sample/model/profile/ProfilesRepositoryFirestoreTest.kt

```diff
@@ -62,22 +59,114 @@ class ProfilesRepositoryFirestoreTest {
   }
 
   @Test
-  fun updateProfile_shouldCallFirestoreCollectionn() {
-    val mockTaskVoid: Task<Void> = Tasks.forResult(null)
+  fun addProfileToDatabase_shouldCallFirestoreToAddUserProfile() {
+    val mockTaskVoid = Tasks.forResult<Void>(null)
     `when`(mockDocumentReference.set(testUser)).thenReturn(mockTaskVoid)
 
-    profileRepositoryFirestore.updateProfile(testUser, onSuccess = {}, onFailure = {})
+    profileRepositoryFirestore.addProfileToDatabase(testUser, onSuccess = {}, onFailure = {})
 
     verify(mockDocumentReference).set(testUser)
   }
 
   @Test
-  fun addProfileToDatabase_shouldCallFirestoreToAddUserProfile() {
-    val mockTaskVoid: Task<Void> = Tasks.forResult(null)
-    `when`(mockDocumentReference.set(testUser)).thenReturn(mockTaskVoid)
+  fun addActivity_shouldCallFirestoreToAddActivity() {
+    `when`(mockDocumentReference.update(eq("activities"), any())).thenReturn(Tasks.forResult(null))
 
-    profileRepositoryFirestore.addProfileToDatabase(testUser, onSuccess = {}, onFailure = {})
+    profileRepositoryFirestore.addActivity("1", "activityId", onSuccess = {}, onFailure = {})
 
-    verify(mockDocumentReference).set(testUser)
+    verify(mockDocumentReference).update(eq("activities"), any())
+  }
+
+  @Test
```

Comment:
Verifies that update is called on Firestore when adding a liked activity.
Uses verify to check the method interaction with Firestore.